### ISSUE TITLE
Relative path to loaded file in same directory its not working(not found), has to be absolute..

### ISSUE THREAD

**Issue by ruthan** [Baron-von-Riedesel/Jemm#20]
I tried this. jload and QPIEMU.dll are both in C:\Drivers\SBEMU directory:

_C:\Drivers\SBEMU\jload QPIEMU.dll_
It was failling - JLoad QPIEMU.dll cannot be open error.. i had to add full patch to QPIEMU.dll too..
_C:\Drivers\SBEMU\jload C:\Drivers\SBEMU\QPIEMU.dll_
I had no more space in my config to add other SBEMU directly to PATH i has 256 chars limit on DOS 7 and 128 on Dos 6..
But i would expected that jload would be clever enough to search this file in own execution directory! its not and its not user friendly, intuitive etc.. 
  If its not how its designed and its feature not bugs, its still better to load file and write some warning that its loaded from local directory than not load it at al..

**Comment by Baron-von-Riedesel** [comment]
> But i would expected that jload would be clever enough to search this file in own execution directory! its not and its not user friendly, intuitive etc..

Yes. There's a tradeoff - surely jload could be made a bit smarter, but this would increase its conventional memory footprint. And the size of this "footprint" may be critical, if jload is invoked by DEVICEHIGH or LH.

**Comment by ruthan** [comment]
Well its all loaded into memory as whole at the start.. and if "loader" (piece of code needed to start it) and runtime (real worker) is one utility and it makes memory footprint higher, maybe its good idea change whole design and make some launcher utility.. which can be quite big.. and some small runtime utility. 

   Other thing is how much bigger would few lines of code make it, i mean its one file search in actual directory and in some if/ else condition, it would be probably nothing..
